api/sixtyfps-cpp/.reuse/dep5

```diff
@@ -0,0 +1,12 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
Hm, do we need this for creates that we don't publish?

Reply:
I did just set this up for any folder containing a Cargo.toml file: Simplest possible approach:-)

Since we need to keep the copyright straight for all our code, that seems like a sensible thing to me.

Reply:
Maybe I'm missing something in my understanding of who uses this. How are these files parsed by reuse and what is reuse's awareness of crates versus just directories?

Reply:
Reuse is blissfully unaware of crates:-) But we want the sources that are uplaoded to crates.io to be scan-able to get the inventory of licenses used. So those need to be self-contained in the tarball that gets sent to crates.io.

Strictly speaking we only need to have the crates we upload separated out. Maybe I should do that... it gets rid of the LICENSES and the .reuse folder, but comes at the small price of needing to create and populate those when we make decide to publish a formerly private crate.

Reply:
I removed this for the non-published crates.